en/docs/learn/design-api/test-api/test-an-api.md

```diff
@@ -0,0 +1,56 @@
+# Test APIs through the publisher test console
+
+**Testing APIs** is trying out the APIs at the publisher itself and make sure whether the functionalities and behaviours are  met before publishing to the gateway for accessibility of the subscribers.
+
+Only the API developers (creator, publisher) are allowed to test the APIs through the test console. The developers can do the basic functional tests such as the mediation policies, make sure whether the request goes as expected to the back-end
+and the proper responses are received, request/response schema validation and check whether the API resources are defined as expected.
+
+The **Test Console** menu/option in the Publisher portal's API details page can be used to initialize the test at the design phase. Once the developer initiates the test by clicking on the initialize test button, the API is transformed to the prototype(testing)
+state and the swagger console is populated with a test-key(token) which prevents the unauthorized users accessing the particular API.
+
+If the API is in the created lifecycle state, the developer can initialize the test.
+
+!!! Note
+    We don't allow testing for the APIs that are in publish state.Hence API developer/publisher has to demote the API to prototype/created
+    state.
+    ![]({{base_path}}/assets/img/learn/publisher-testconsole-publishstate.png)
+
+Let's see how to use the Publisher test Console to test an API.
+
+The examples here use the `PizzaShack` REST API, which was created in [Create a REST API]({{base_path}}/learn/design-api/create-api/create-a-rest-api/) .
+
+1.  Sign in to the API Publisher `https://<hostname>:9443/publisher` (e.g., `https://localhost:9443/publisher` ). Upon signing in, the list of APIs in the API Publisher is listed. Please refer [create an API guide](/learn/design-api/create-api/create-a-rest-api/) to create a new API.
+
+     The list of APIs in the API Publisher appears. If there are no APIs created, [create an API]({{base_path}}/learn/design-api/create-api/create-a-rest-api/) before starting.
```

Comment:
since this sentence "The list of APIs in the API Publisher appears" is duplicated, shall we remove it here?